modules/test/playwright/pages/commerce/commerceChannelDefaultsPage.ts

```diff
@@ -109,5 +112,16 @@ export class CommerceChannelDefaultsPage {
 			exact: true,
 			name: 'Save',
 		});
+		this.channelEntryHeader = (channelEntryHeaderName: string) => {
+			return page.locator(`h3:has-text("${channelEntryHeaderName}")`);
```

Comment:
try with `page.getByRole('heading', {name: channelEntryHeaderName})`

Reply:
also, is it correct that this is from `page `and not from `this.channelEntry` like the addButton?

Reply:
Yeah, it should be ok, at least it was this way in the old poshi test. I can't see any particular reason to change it.
If the headers are there that's good, that's all we want to check.

Reply:
Nevertheless, changing it to page.getByRole('heading', {name: channelEntryHeaderName}).

Reply:
On the other hand, h3 is more specific than heading? heading could be h2 for instance?